.github/workflows/ci.yaml

```diff
@@ -27,14 +27,14 @@ jobs:
 
       - name: Build Docker Image
         run: |
-          docker build -t "ghcr.io/neuro-inc/mlflow:${{ github.event.release.tag_name }}" .
+          docker build -t "ghcr.io/apolo-actions/mlflow:${{ github.event.release.tag_name }}" .
```

Comment:
let's call tests before releasing the image, wdyt?

Reply:
I believe I added this dependency with this line in the previous PR. Does that solve it? https://github.com/apolo-actions/mlflow/blob/6da6162402409778f6486e4fb7ee08a352205e62/.github/workflows/ci.yaml#L15